gnosis/safe-react
# fix: historically estimate `maxPriorityFeePerGas`

## What it solves
Resolves #3393

## How this PR fixes it
The 50th percentile of the latest block is retrieved from `getFeeHistory` and `maxPriorityFeePerGas`/`maxFeePerGas` are parses/returned from it. The `maxPriorityFeePerGas` is then clamped between `0.001` and `maxFeePerGas - 1` if necessary.

Full test coverage is included.

## How to test it
- Execute a transaction and observe that:
  - `maxFeePerGas` is higher than `maxPriorityFeePerGas`
  - execution is successful
